### ISSUE TITLE
Loader doesn't detect stylesheets if using babel with import

### ISSUE THREAD

**Issue by leoasis** [js-next/react-style-webpack-plugin#14]
When using `import` with babel it fails to detect the `StyleSheet.create` call because it gets transformed by the babel loader, and the current loader is just using a regexp that fails to test in that case.

To add some guidance, using the babel repl for the following source:
```js
import StyleSheet from 'react-style';

StyleSheet.create({
  foo: {
    color: 'red'
  }
});
```

gives the following output, which is what the react-style loader sees:

```js
'use strict';

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

var _reactStyle = require('react-style');

var _reactStyle2 = _interopRequireDefault(_reactStyle);

_reactStyle2['default'].create({
  foo: {
    color: 'red'
  }
});
```

Using the regular `require` works correctly, since babel doesn't alter that.

Perhaps I should just put the react-style loader before the babel one?
